src/app/clusters/time-synchronization-server/DefaultTimeSyncDelegate.cpp

```diff
@@ -78,3 +78,13 @@ void DefaultTimeSyncDelegate::UTCTimeAvailabilityChanged(uint64_t time)
 {
     // placeholder implementation
 }
+
+void DefaultTimeSyncDelegate::TrustedTimeSourceAvailabilityChanged(bool available, GranularityEnum granularity)
+{
+    // placeholder implementation
```

Comment:
how about making them {} instead of `=0` in the base class to not need empty implementations always?

Reply:
sounds good